### DISCUSSION TITLE
Adding x86_64 support

### DISCUSSION THREAD

**Discussion by hmtheboy154** [bromite/bromite#1250] [Ideas]
Hello, I'm asking if you guys can adding x86_64 support for both Bromite and Bromite Webview ? Since there are projects like Android-x86 and BlissOS that ship with x86_64. When I put Bromite Webview into the system a lot of apps crash since it trying to look for x86_64 version. 
Also it would be great if I can use Bromite in x86_64. 

If you are looking for someone to test, I can help you with that.

**Comment by ilhan-athn7** [comment]
I agree, it would be nice.
